stats/record.go

```diff
@@ -17,12 +17,17 @@ package stats
 
 import (
 	"context"
+	"log"
+	"os"
 
 	"go.opencensus.io/stats/internal"
 	"go.opencensus.io/tag"
 )
 
+var debug bool
+
 func init() {
+	debug = os.Getenv("OC_STATS_DEBUG") != ""
```

Comment:
Let's document this perhaps?

Reply:
Don't think it should be a documented feature of the library so I removed it instead.